src/Surfnet/ServiceProviderDashboard/Domain/Entity/ManageEntity.php

```diff
@@ -350,4 +353,18 @@ public function asArray(): array
         $data += $this->protocol->asArray();
         return $data;
     }
+
+    public function isStatusPublicationRequested()
+    {
+        return $this->getStatus() === Constants::STATE_PUBLICATION_REQUESTED;
+    }
```

Comment:
This method is only used within this class and in the unit test. That means that it (c|sh)ould be private. In the test you could simply get the status and verify if it equals `Constants::STATE_PUBLICATION_REQUESTED`

Reply:
Only used here so private is a better option